platform/common/core/generic/vault/fver/version.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package fver
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/hyperledger-labs/fabric-smart-client/pkg/utils/errors"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
+)
+
+var zeroVersion = []byte{0, 0, 0, 0, 0, 0, 0, 0}
+
+func IsEqual(a, b driver.RawVersion) bool {
+	if bytes.Equal(a, b) {
+		return true
+	}
+	if len(a) == 0 && bytes.Equal(zeroVersion, b) {
+		return true
+	}
+	if len(b) == 0 && bytes.Equal(zeroVersion, a) {
+		return true
+	}
+	return false
+}
+
+func ToBytes(Block driver.BlockNum, TxNum driver.TxNum) []byte {
+	buf := make([]byte, 8)
+	binary.BigEndian.PutUint32(buf[:4], uint32(Block))
```

Comment:
Don't we run into issues here if fabric is used? Block height in Fabric is still uint64? I am wondering if we need to fix the version size at all?